### ISSUE TITLE
Clicking download again will not pop up or download.

### ISSUE THREAD

**Issue by drc5521** [kiwibrowser/src.next#606]
**Describe the bug**
When I click a download button for the first time, there will be a download prompt. Click Cancel and click the download button again. There will be no pop-up and download. You must refresh the page to appear again.

**Expected behavior**
You don't need to refresh again to download.

**Smartphone (please complete the following information):**
 - Device: [e.g. OnePlus 7]HONOR X10
 - Android Version : [e.g. A11 ]A10
 - Kiwi Browser Version [ e.g. 90.0.4577.17 ]101.0.4951.48

**Additional context**

**Comment by gnadelwartz** [comment]
Kiwi itself has no "download button" so I guess you mean a button on a website, without a website having this behavior it's not possible to reproduce and fix your problem.

**Comment by drc5521** [comment]
> Kiwi itself has no "download button" so I guess you mean a button on a website, without a website having this behavior it's not possible to reproduce and fix your problem.

Yes,you are right. It's on a website. It may be a download button or link. eg. github Releases.

**Comment by gnadelwartz** [comment]
Ok, I tested  GitHub download apk an source zip before my first answer and it worked multiple times without refreshing. In addition there is no cancel button on the download popup, you can only swipe to pause.

Have a look in menu under downloads if your download is paused instead of cancelled, plus check if problem persists with AdBlock and all extensions disabled.

**Comment by drc5521** [comment]
> Ok, I tested GitHub download apk an source zip before my first answer and it worked multiple times without refreshing. In addition there is no cancel button on the download popup, you can only swipe to pause.
> 
> Have a look in menu under downloads if your download is paused instead of cancelled, plus check if problem persists with AdBlock and all extensions disabled.

I close all extensions.Here is a video that you can watch.
https://user-images.githubusercontent.com/55195642/166867041-01fac655-f348-4d06-b20e-afe75c2f35e1.mp4

**Comment by gnadelwartz** [comment]
This popup with "download later" does not appear in my kiwi installation and I've never seen. Do you eventually have a download manager installed?

**Comment by gnadelwartz** [comment]
I have to applogize, now teh poup appers,nevermind before .. lets see if it works only once also for me ...

**Comment by drc5521** [comment]
> This popup with "download later" does not appear in my kiwi installation and I've never seen. Do you eventually have a download manager installed?

yeah,I install ADM,but I never used it.
![Screenshot_20220508_192105_com kiwibrowser browser](https://user-images.githubusercontent.com/55195642/167294024-2a3bc226-de35-496e-8294-0e89fc8e2822.png)

**Comment by drc5521** [comment]
> I have to applogize, now the poup appers,nevermind before .. lets see if it works only once also for me ...
> 
> ... nevertheless, I can klick on downloadlionk and select cancel in (new) popup many times without need to reload.

It doesn't matter. We can discuss it as much as we like.I installed the latest build, and there is still this problem.
